build/raise-version/raise.sh

```diff
@@ -62,13 +62,15 @@ function raiseVersionOfOurRepos {
     "git@github.com:axonivy/inscription-client.git"
     "git@github.com:axonivy/core.git"
     "git@github.com:axonivy/thirdparty-libs.git"
+    "git@github.com:axonivy/theia-ide.git"
+    "git@github.com:axonivy/vscode-extensions.git"
   )
   message="Raise version to ${newVersion}"
   runRepoUpdate 'updateSingleRepo' ${repos[@]}
 }
 
 function updateSingleRepo {
-  .ivy/raise-version.sh ${newVersion}
+  .ivy/raise-version.sh ${newVersion} ${dryRun}
```

Comment:
@alexsuter note that I plan to pass dryrun as param to the raise script

Reply:
hmm. I guess this is fine, but are you sure you need it, what are you doing with this param in your raise-version.sh script? can you point me to that?

Reply:
https://github.com/axonivy/vscode-extensions/blob/master/.ivy/raise-version.sh
`yarn install` will lead to `Error: Couldn't find any versions for "@axonivy/inscription-protocol" that matches "~20.0.0-next"`

Reply:
This will work, if you set dryRun=false when you really want to make raise-version?
If yes then I'm fine.

I would assign the variable $2 to a meaningful variable
e.g.
```
dryRun=$2
```

P.S. You can't give yarn a param to say "doNotCheckVersions" ?